x/committee/testutil/types.go

```diff
@@ -1,11 +1,24 @@
 package testutil
 
 import (
+	"testing"
+
+	"github.com/cosmos/cosmos-sdk/codec"
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/gogo/protobuf/proto"
+	"github.com/stretchr/testify/assert"
 )
 
 // Avoid cluttering test cases with long function names
 func I(in int64) sdk.Int                    { return sdk.NewInt(in) }
 func D(str string) sdk.Dec                  { return sdk.MustNewDecFromStr(str) }
 func C(denom string, amount int64) sdk.Coin { return sdk.NewInt64Coin(denom, amount) }
 func Cs(coins ...sdk.Coin) sdk.Coins        { return sdk.NewCoins(coins...) }
+
+func AssertProtoMessageJSON(t *testing.T, cdc codec.Codec, expected proto.Message, actual proto.Message) {
```

Comment:
is there a better place to put assertion helpers like this that might be used by other modules as well?

Reply:
Not currently but it would be nice to start one. I had a quick look at other big go repos and couldn't see any standard people use. Maybe `/testutil` or `/test`?